feature/news/src/main/java/com/teamwable/news/notice/NewsNoticeRoute.kt

```diff
@@ -45,25 +45,23 @@ fun NewsNoticeScreen(
     notices: List<NewsInfoModel>,
     navigateToDetail: (NewsInfoModel) -> Unit
 ) {
-    Box(modifier = Modifier.fillMaxSize()) {
-        LazyColumn(modifier = Modifier.fillMaxSize()) {
-            items(notices) { notice ->
-                NewsNoticeItem(context, notice, navigateToDetail)
-                HorizontalDivider(
-                    thickness = 1.dp,
-                    color = WableTheme.colors.gray200,
-                )
-            }
+    LazyColumn(modifier = Modifier.fillMaxSize()) {
+        items(items = notices, key = { item -> item.newsId }) { notice ->
+            NewsNoticeItem(context, notice, navigateToDetail)
+            HorizontalDivider(
+                thickness = 1.dp,
+                color = WableTheme.colors.gray200,
+            )
         }
     }
 }
 
 @Composable
-fun NewsNoticeEmptyScreen() {
+fun NewsNoticeEmptyScreen(emptyTxt: Int) {
```

Comment:
```suggestion
fun NewsNoticeEmptyScreen(@StringRes emptyTxt: Int) {
```
만 붙여주세요 뒤에 pr에서 하셔도 됩니다 짱!

Reply:
넹~